testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDeleteCompatibility.java

```diff
@@ -276,6 +276,31 @@ public void testDeletionVectors(String mode)
         }
     }
 
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
```

Comment:
follow-up: Please add as well a test resource similar to `plugin/trino-delta-lake/src/test/resources/databricks122/deletion_vectors`  containing a table with partitioning columns.